app/safe/cmd/main.go

```diff
@@ -15,12 +15,20 @@ import (
 
 	"github.com/vmware-tanzu/secrets-manager/app/safe/internal/bootstrap"
 	"github.com/vmware-tanzu/secrets-manager/app/safe/internal/server"
-	"github.com/vmware-tanzu/secrets-manager/core/log"
+	"github.com/vmware-tanzu/secrets-manager/core/crypto"
+	log "github.com/vmware-tanzu/secrets-manager/core/log/std"
 	"github.com/vmware-tanzu/secrets-manager/core/probe"
 )
 
 func main() {
-	id := "VSECMSAFE"
+	id, err := crypto.RandomString(8)
```

Comment:
Create a random id and assign it to the context to pass through the chain of requests and responses.

this id will preserve through the sequence of actions related to this particular command execution.